about.html

```diff
@@ -0,0 +1,184 @@
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="maindiv">
+      <header>
+        <nav class="nav">
+          <div class="mobile-nav mobile">
+            <button class="hamburger-menu">
+              <i class="fa-sharp fa-solid fa-bars fa-3x" id="hamburgericon"></i>
+            </button>
+          </div>
+        </nav>
+        <div id="myNav" class="overlay mobile">
+            <div class="mobile-menu-navbar">
+              <a href="javascript:void(0)" class="closeicon"><i class="closeIcon material-icons  fa-3x"
+                  id="close-icon">close</i></a>
+            </div>
+            <div class="overlay-content">
+              <a href="index.html" onclick="closeNav()">Home</a>
+              <a href="#project-title" onclick="closeNav()">Program</a>
+              <a href="#about" onclick="closeNav()">Join</a>
+              <a href="#form" onclick="closeNav()">Sponsor</a>
+              <a href="#form" onclick="closeNav()">News</a>
+            </div>
+          </div>
+          <div class="desktop">
+            <div class="desktopmenu">
+              <div>
+                <img 
+                  src="./assets/img/gamerzonefooter.png"
+                  alt="project1"
+                  class="desktopmenuimg"
+                />
+              </div>
+              <div class="menuitemsflex">
+                <a href="index.html">Home</a>
+                <a href="#project-title">Program</a>
+                <a href="#about">Join</a>
+                <a href="#form">Sponsor</a>
+                <a href="#form">News</a>
+                <button class="menu-button">BOOK EVENT</button>
+              </div>
+            </div>
+          </div>
+      </header>
+      <main>
+        <section class="mobile-view" id="">
+          <div>
+            <p class="firstheader center">"Hello Gamers Around The Globe"</p>
+            <h1 class="secondheader2 center">GAMERSZONE ANNUAL EVENT MAINZ 2022</h1>
+          </div>
+          <div class="paragraph-mobile2 center">
+            <p>
+              An annual gathering of all the best gamers every years, meet, challenge
+              learn from the leading Gamers and added tips how to make money by being a gamer.
+            </p>
+          </div>
+          <div>
+            <p class="paragraph-mobile-about center">
+              Please do well to contact us using this email.<br />
+              <b>fitzgeraldnsong@gmail.com</b>
+            </p>
+          </div>
+        </section>
```

Comment:
- [ ] `required`

- [ ] Please adjust the typography, Fonts and spacing according to the design
- [ ] i have highlighted things which needs to be adjusted 
- [ ] please see screenshots below for comparisons

- [ ] Your deign
![image](https://user-images.githubusercontent.com/66230306/210710697-764a7c8a-907f-4726-9ce6-dcce80828e31.png)

- [ ]  Behance design
![image](https://user-images.githubusercontent.com/66230306/210709975-ad85a28b-85c7-484e-b842-d6392e202ab2.png)